core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -401,14 +401,14 @@ Executor startStopExecutor() {
     /**
      * Returns the {@link ChannelOption}s and their values of {@link Server}'s server sockets.
      */
-    public Map<ChannelOption<?>, ?> channelOptions() {
+    public Map<ChannelOption<?>, Object> channelOptions() {
         return channelOptions;
     }
 
     /**
      * Returns the {@link ChannelOption}s and their values of sockets accepted by {@link Server}.
      */
-    public Map<ChannelOption<?>, ?> childChannelOptions() {
+    public Map<ChannelOption<?>, Object> childChannelOptions() {
```

Comment:
This is maybe a breaking change. Could we revert this and let the caller down-cast explicitly?